### PR TITLE
[config] Add ServiceAccount manifest

### PR DESCRIPTION
**Pull request by Fedosin**
When I deploy the operator with `make deploy`, I get next error from WMCO deployment:

```txt
error looking up service account openshift-windows-machine-config-operator/windows-machine-config-operator:
      serviceaccount "windows-machine-config-operator" not found'
```

This commit adds a manifest for the missing service account. After that everything installs well.

### CODE REVIEW

**Comment by mansikulkarni96** [comment]
Thanks for fixing this @Fedosin 
This was removed as part of a service_account duplication manifest issue. Tested it locally, works well as long as we do not add the service account file to the manifests directory that gets created with this when we do `make bundle`

/lgtm

File: config/rbac/service_account.yaml
Comment (aravindhp) [inline config/rbac/service_account.yaml:2]: This is only required if you are adding multiple "kinds".

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for fixing this, @Fedosin. Please add a `[config]` prefix to the commit and PR headers.

**Comment by sebsoto** [comment]
/approve
As this is manually tested and linting PR is in progress

**Comment by aravindhp** [comment]
/hold

Azure cluster bring up is consistently failing

**Comment by aravindhp** [comment]
/hold

until we figure out why the override for the cluster approver is not taking effect causing the CSR validation test to fail

**Pull request merged**
